pkgs/tools/typesetting/soupault/camomile-2_0_0.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, ocamlPackages
+}:
+
+# There was little interest in upstreaming this package as a whole, at
+# least for now. See Microsoft GitHub Issue:
+# https://github.com/NixOS/nixpkgs/issues/224318
+
+let
+  pname = "camomile";
+  version = "2.0.0";
+in
+ocamlPackages.buildDunePackage {
+  inherit pname version;
```

Comment:
```suggestion
ocamlPackages.buildDunePackage rec {
  pname = "camomile";
  version = "2.0.0";
```

Reply:
Aren't there consequences to `rec` + `override*` functions? I've been bitten in the past by it like https://stackoverflow.com/questions/56558483/nix-override-docheck-in-pybind11

Reply:
👍‍